task2/rest-api-basics/domain/src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -0,0 +1,124 @@
+package com.epam.esm.entity;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Objects;
+import java.util.Set;
+
+public class GiftCertificate implements Serializable {
+    private static final long serialVersionUID = 7468316931994434280L;
+
+    private Long id;
```

Comment:
@igor-zhuravlev The same (Certificate, Tag) id and name fields. Could be moved to abstract class.